backend/apps/predictions/ml_engine.py

```
@@ -0,0 +1,98 @@
+            gbr = GradientBoostingRegressor(n_estimators=20, random_state=42)
+            gbr.fit(X_synthetic, y_synthetic)
+
+            input_vec = np.array([[
+                features["total_lines"],
+                features["changed_files"],
+                features["current_workload"],
+                features["activity_score"],
+                features["avg_response_time_hours"]
+            ]])
+            predicted_delay = float(gbr.predict(input_vec)[0])
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)